sensors/sensors.go

```diff
@@ -15,10 +16,18 @@ import (
 	"go.viam.com/rdk/pointcloud"
 	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/spatialmath"
+	rdkutils "go.viam.com/rdk/utils"
 	"go.viam.com/rdk/utils/contextutils"
 	goutils "go.viam.com/utils"
 )
 
+const (
+	replayTimeTolerance         = 10 // Milliseconds
```

Comment:
[nit] I think the var name should contain the time unit, e.g.:

```suggestion
	replayTimeToleranceMsec         = 10 // Milliseconds
```

(not sure if `Msec` is the convention we follow)